### DISCUSSION TITLE
What do you do when you miss `ct`?

### DISCUSSION THREAD

**Discussion by pkazmier** [nvim-mini/mini.nvim#932] [Q&A]
If I miss my intended target with `dt` because of extra matching letters between my cursor and target, with mini.jump I'm able to easily see the extra characters (as they are underlined) and simply dot-repeat until I reach my target. For example, if I have the following text in a buffer with my cursor at the start of the line:

```
The weather is nice outside today.
```

To delete up to "today", I'd likely quickly press `dtt`, but then realize there was a letter "t" in both "weather" and "outside". No big deal as I'd dot-repeat two times until I reach "today".

I'm curious, however, what would you do if instead of `dt` you wanted to change text with `ct` in this scenario? For me, after pressing `ctt`, I'd curse because I missed my target, then escape, undo, and use mini.jump2d to reach "today" directly. Sure, I know that I could have used mini.jump2d in the first place. Likewise, I could have used `c/today`.

But I generally reach for `ct` first (less keystrokes in the happy path) unless I happen to notice multiple instances of my target character. Unfortunately, I frequently don't notice those resulting in my frustration and thus my question. Is there a better way of recovering from a missed `ct` that has eluded me?

**Comment by pkazmier** [comment]
I suppose upon the first miss with `ct`, I could just hit escape, dot repeat 2x, the press `I` to get back into insert mode. Not sure why I don't have the baked into muscle memory. Nonetheless, just wanted to make sure there is not a better way of recovering.

**Comment by echasnovski** [comment]
Unfortunately, I don't really have some kind of profound answer here other than "do what works for you".

For me personally (I think...), I usually prefer `c` operator to be used with textobjects (`ciw`, `cinl)`, `cgn`, etc.) or when I *need* dot-repeatability. While with motion it is more `dt)` followed by `i` *or* `vt)` followed by `c`.

**Reply by pkazmier** [reply]
That sounds like good advice in general. Thanks for confirming that I was not missing something obvious.
